modules/navigation/src/main/java/com/sygic/maps/module/navigation/viewmodel/NavigationFragmentViewModel.kt

```diff
@@ -99,11 +98,19 @@ class NavigationFragmentViewModel internal constructor(
     @LayoutRes
     val signpostLayout: Int
     val signpostEnabled: MutableLiveData<Boolean> = MutableLiveData(SIGNPOST_ENABLED_DEFAULT_VALUE)
+    val infobarEnabled: MutableLiveData<Boolean> = MutableLiveData(INFOBAR_ENABLED_DEFAULT_VALUE)
     val previewControlsEnabled: MutableLiveData<Boolean> = MutableLiveData(PREVIEW_CONTROLS_ENABLED_DEFAULT_VALUE)
 
+    val leftInfobarButtonWrapper = InfobarButtonWrapper()
```

Comment:
toto sa vola z xml (bindingu) ze? v zasade by to v nazve premennej nemuselo mat `Wrapper`. Lepsie by bolo iba `leftInfobarButton`

Reply:
jj, ok. odstraním